src/chrome/content/rules/Fasthosts.xml

```diff
@@ -39,7 +39,7 @@
 	<rule from="^http://(?:www\.)?fasthosts\.co\.uk/"
 		to="https://www.fasthosts.co.uk/" />
 
-	<rule from="^http://secure(\d|1[0-3-9]|2[0-4]|5[26])\.prositehosting\.co\.uk/"
```

Comment:
I'm not sure what the original `[0-3-9]` was intended to match; the replacement pattern is not what python currently matches (0-3, or dash, or 9).

Reply:
@thatch This is correct. This will match any of `['0', '1', '2', '3', '9', '-']`.

Reply:
@pipboy96 the replacement on the line below doesn't escape the `-` in `-9` though, so it changes the meaning.